<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>M/S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>KM/H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor6">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo6">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MILHAS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>QUILOMETROS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor1">
       <property name="maximum">
        <number>1000000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calculo">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>HORA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>SEGUNDOS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor3">
       <property name="maximum">
        <number>10000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo3">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>TONELADA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>QUILOGRAMA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor7">
       <property name="maximum">
        <number>10000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo7">
       <property name="text">
        <string>CALULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>QUILOGRAMA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>TONELADA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor8">
       <property name="maximum">
        <number>10000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo8">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>QUILOMETROS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>MILHAS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor2">
       <property name="maximum">
        <number>100000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo2">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>KM/H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultado5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>M/S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="valor5">
         <property name="maximum">
          <number>10000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculo5">
         <property name="text">
          <string>CALCULAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>SEGUNDOS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resultado4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>HORA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="valor4">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculo4">
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>20</y>
      <width>139</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair do Programa</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>168</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Valores das Conversões</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="gravar">
           <property name="text">
            <string>Gravar Arquivo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Abrir Arquivo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCONVERS_O_DISTANCIA">
    <property name="title">
     <string>CONVERSÃO DISTANCIA</string>
    </property>
    <addaction name="actionQuilometros_MILHAS"/>
    <addaction name="actionMilhas_Quilometros"/>
   </widget>
   <widget class="QMenu" name="menuCONVERS_O_DE_TEMPPO">
    <property name="title">
     <string>CONVERSÃO DE TEMPO</string>
    </property>
    <addaction name="actionHora_Segundos"/>
    <addaction name="actionSegundos_Hora"/>
   </widget>
   <widget class="QMenu" name="menuCONVERS_O_DEVELOCIDADE">
    <property name="title">
     <string>CONVERSÃO DEVELOCIDADE</string>
    </property>
    <addaction name="actionKm_h_M_s"/>
    <addaction name="actionM_s_Km_h"/>
   </widget>
   <widget class="QMenu" name="menuCONVERS_O_DE_MASSA">
    <property name="title">
     <string>CONVERSÃO DE MASSA</string>
    </property>
    <addaction name="actionTonelada_Quilograma"/>
    <addaction name="actionQuilograma_Tonelada"/>
   </widget>
   <addaction name="menuCONVERS_O_DISTANCIA"/>
   <addaction name="menuCONVERS_O_DE_TEMPPO"/>
   <addaction name="menuCONVERS_O_DEVELOCIDADE"/>
   <addaction name="menuCONVERS_O_DE_MASSA"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuilometros_MILHAS">
   <property name="text">
    <string>Quilometros-Milhas</string>
   </property>
  </action>
  <action name="actionMilhas_Quilometros">
   <property name="text">
    <string>Milhas-Quilometros</string>
   </property>
  </action>
  <action name="actionHora_Segundos">
   <property name="text">
    <string>Hora-Segundos</string>
   </property>
  </action>
  <action name="actionSegundos_Hora">
   <property name="text">
    <string>Segundos-Hora</string>
   </property>
  </action>
  <action name="actionKm_h_M_s">
   <property name="text">
    <string>Km/h-M/s</string>
   </property>
  </action>
  <action name="actionM_s_Km_h">
   <property name="text">
    <string>M/s-Km/h</string>
   </property>
  </action>
  <action name="actionTonelada_Quilograma">
   <property name="text">
    <string>Tonelada-Quilograma</string>
   </property>
  </action>
  <action name="actionQuilograma_Tonelada">
   <property name="text">
    <string>Quilograma-Tonelada</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
